src/cart/src/cart/crt_ctl.c

```diff
@@ -171,7 +146,7 @@ crt_hdlr_ctl_get_uri_cache(crt_rpc_t *rpc_req)
 		D_GOTO(out, 0);
 
 	out_args->cguc_grp_cache.ca_arrays = uri_cache.grp_cache;
```

Comment:
It seems like in the -DER_OVERFLOW case here you could still return the data, along with the return code and then the client presumably need to act appropriately.  I'm not sure what this RPC is actually used for so so don't know if that would be useful.

Reply:
Modified code to send truncated array.